packages/browser/src/index.ts

```diff
@@ -1,8 +1,19 @@
-export { BacktraceReport, BacktraceStackTraceConverter } from '@backtrace/sdk-core';
-export { JavaScriptEngine } from '@backtrace/sdk-core/lib/model/data/JavaScriptEngine';
-export { BacktraceStackFrame } from '@backtrace/sdk-core/lib/model/data/BacktraceStackTrace';
-export * from '@backtrace/sdk-core/lib/model/attachment';
+export {
+    AttributeType,
+    BacktraceAttachment,
+    BacktraceData,
+    BacktraceReport,
+    BacktraceStackFrame,
+    BacktraceStackTraceConverter,
+    BacktraceStringAttachment,
+    BacktraceUint8ArrayAttachment,
+    BreadcrumbLogLevel,
+    BreadcrumbsEventSubscriber,
+    BreadcrumbsManager,
+    BreadcrumbType,
+    JavaScriptEngine,
+} from '@backtrace/sdk-core';
```

Comment:
maybe just export everything?
```suggestion
export * from '@backtrace/sdk-core';
```

Reply:
unfortunately there might be a naming issue - example: backtraceConfiguration